src/npc.cpp

```diff
@@ -2315,4 +2315,97 @@ void npc_chatbin::add_new_mission( mission *miss )
     missions.push_back( miss );
 }
 
+epilogue::epilogue()
+{
+    id = "NONE";
+    group = "NONE";
+    is_unique = false;
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("           ###### #### ####   ######    ####    ###   #### ######           ");
+    lines.push_back("            ##  #  ##   ##     ##  #     ##    ## ## ##  # # ## #           ");
+    lines.push_back("            ####   ##   ##     ####      ##    ## ## ####    ##             ");
+    lines.push_back("            ##     ##   ##     ##        ##    ## ##   ###   ##             ");
+    lines.push_back("            ##     ##   ## ##  ## ##     ## ## ## ## #  ##   ##             ");
+    lines.push_back("           ####   #### ###### ######    ######  ###  ####   ####            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+    lines.push_back("                                                                            ");
+}
+
+epilogue_map epilogue::_all_epilogue;
+
+void epilogue::load_epilogue(JsonObject &jsobj)
+{
+    epilogue base;
+    base.id = jsobj.get_string("id");
+    base.group = jsobj.get_string("group");
+    base.is_unique = jsobj.get_bool("unique", false);
+    base.lines.clear();
```

Comment:
Nitpicking: this `clear` is useless, `base` is a new object, so its member `lines` is also a new object, and vectors are created empty, so `base.lines` *is* already empty, there is no need to clear it.